frappe/public/js/frappe/form/quick_entry.js

```diff
@@ -161,9 +161,13 @@ frappe.ui.form.QuickEntryForm = class QuickEntryForm {
 
 			if (data) {
 				me.dialog.working = true;
-				me.dialog.set_message(__("Saving..."));
 				me.insert().then(() => {
-					me.dialog.clear_message();
+					let messagetxt = __("Created new {0} '{1}'", [__(me.doctype), this.doc.name]);
+					me.dialog.animate = "slow";
+					me.dialog.hide();
+					setTimeout(function () {
+						frappe.show_alert({ message: messagetxt, indicator: "green" }, 3);
+					}, 1000);
```

Comment:
```suggestion
					}, 500);
```
should be enough? currently, the delay is noticeable and is giving kind of a "disconnected feel"

Reply:
You are probably right. I will do so.